todoman/model.py

```diff
@@ -322,7 +322,10 @@ def save(self, list_=None):
         list_ = list_ or self.list
         path = os.path.join(list_.path, self.filename)
         assert path.startswith(list_.path)
-
+        y = self.todo.get('SEQUENCE')
```

Comment:
I'd rather this variable be called `seq` or `sequence`, rather than `y` (eg: something more descriptive).